src/main/java/frc/robot/Constants.java

```diff
@@ -34,28 +35,59 @@
 
     public static final int kDBLeftJoystickAxisX = 0;
     public static final int kDBLeftJoystickAxisY = 1;
     public static final int kDBRightJoystickAxisX = 2;
     public static final int kDBRightJoystickAxisY = 3;
 
-    // Intake Contstants 
+    // Intake Contstants
     public static final int kIntakeMotorOneID = 11;
     public static final int kIntakeMotorTwoID = 12;
     public static final double kIntakeMotorSpeed = 0.25;
 
-    //Hopper Constants
+    // Hopper Constants
     public static final int kHopperMotorThreeID = 13;
     public static final double kHopperMotorSpeed = 0.25;
-    
-    //Joystick Constants
+
+    // Joystick Constants
     public static final int kPortNumber = 0;
     public static final int kJoystickButtonNumberOne = 1;
     public static final int kJoystickButtonNumberTwo = 2;
-    public static final int kAButton = 1;           // Button for hopper
-    public static final int kRightBumperButton = 5; // Button for intake 
-    public static final int kLeftBumperButton = 6;  // Button to reverse intake
+    public static final int kAButton = 1; // Button for hopper
+    public static final int kRightBumperButton = 5; // Button for intake
+    public static final int kLeftBumperButton = 6; // Button to reverse intake
+    public static final int kXbutton = 3; // Button for Shooter
+    public static final int kDownbutton = 4; // Button for shooter mode
+    public static final int kUpbutton = 5; // Button for shooter mode
+
+    // Distance calculation constants
+    public static final double kLLHeight = 1.5; // Height of the limelight in ft from the carpet
+    public static final double kLLAngle = 30.0; // Angle that the limelight is mounted at
+
+    public static final double kGoalHeight = 8.8; // Height of the goal in ft from the carpet
+    // Preset aim constants
+    public static final int kLOWRPM = 0; // RPM that the LOW aimMode winds to
+    public static final double kLOWAngle = 0.0; // Angle that the LOW aimMode adjusts to
+    public static final int kLAUNCHRPM = 0; // RPM that the LAUNCH aimMode winds to
+    public static final double kLAUNCHAngle = 0.0; // Angle that the LAUNCH aimMode adjusts to
+    public static final int kTARMACRPM = 0; // RPM that the TARMAC aimMode winds to
+    public static final double kTARMACAngle = 0.0; // Angle that the TARMAC aimMode adjusts to
+
+    // Shooter Domain
+    public static final int KShooterID = 1; // ID of the shooter
+    public static final int KHoodID = 2; // ID of the hood;
+    public static final double KShooterHeight = 3;
+    public static final double KHighHeight = 8.8;
+    public static final double KLowHeight = 5.8;
+    public static final double KAirboneTime = 1.2;
+    public static final double KGravity = 32.0;
+    public static final double KGearRatioIn = 34;
+    public static final double KGearRatioOut = 14;
+    public static final double KGearDiametter = 4; // Gear diametter in inches
```

Comment:
diameter has one t.  Please change to make it easier to search for the constant.